mne/source_space.py

```diff
@@ -1182,6 +1182,54 @@ def vertex_to_mni(vertices, hemis, subject, subjects_dir=None, mode=None,
     return apply_trans(xfm['trans'], data)
 
 
+##############################################################################
+# Volume to MNI conversion
+
+
+@verbose
+def aseg_vertex_to_mni(vertices, subject, mri_head_t, subjects_dir=None,
+                       verbose=None):
+    """Convert the array of vertices for a hemisphere to MNI coordinates.
+
+    Parameters
+    ----------
+    vertices : n_vertices x 3 array of float
```

Comment:
If these are in the head coordinate system, `pos` would be a better name. `vertices` we try to reserve for vertices on a FreeSurfer mesh.

Reply:
done! 
Since `aseg_vertex_to_mni` works also for vertices in the cortex surface, do we have to change
the name?

Reply:
Well it doesn't work for "vertices" in the sense that we typically mean "vertices" in mne. "vertices" in MNE refer to indices into a subject's Freesurfer surface mesh, not `(n, 3)` arrays of the coordinates of those points.

This function as far as I can tell will take any point in head coords and convert it to MNI coords. So the name is a bit odd. Maybe it should be `head_to_mni`? @agramfort WDYT?

Reply:
Yes, also to me now `aseg_vertex_to_mni` is a weird name now. 
`head_to_mni` sounds good.